templates/index.html

```diff
@@ -1,9 +1,95 @@
-<title>Community website</title>
-<body>
-<ul>
-    <li><a href="/gci/">Google Code-in</a>
-    <li><a href="/activity/">GitHub activity</a>
-    <li><a href="/twitter/">Twitter Feed</a>
-    <li><a href="/log/">Logs</a>
-</ul>
+{% extends 'includes.html' %}
+
+{% block content %}
+    <div class="section no-pad-bot" id="index-banner">
+        <div class="container">
+            <h1 class="header center  col-test">Welcome</h1>
+            <div class="row center">
+                <h5 class="header col s12 light">Hello, world. You are at the {{ org_name }} community website.</h5>
+                <h4 class="header col s12 light">Meet awesome mentors</h4>
```

Comment:
this is not how we would do a mentors app.  please remove, and let that be achieved in the relevant issue.